### ISSUE TITLE
Version 4.0.0 is missing in git

### ISSUE THREAD

**Issue by l-lotz** [solarkennedy/puppet-consul#445]
The current version on Puppet Forge is "4.0.0" but the current master is at "3.4.2". The Readme on Puppet Forge also contains a newer compatibility table.

**Comment by bastelfreak** [comment]
Hey @solarkennedy, how did you do the last release? Can you add a tag for the approriate commit?

**Comment by solarkennedy** [comment]
Er, apparently I did it wrong (again) :(

I'm using puppet blacksmith, but I'm just not doing it right. Let me try again.

**Comment by solarkennedy** [comment]
Whoops, I didn't push. Everything is in sync now.
